action.js

```diff
@@ -59,10 +63,12 @@ module.exports = class {
     const conflictResolutionStart = [
       `Merge branch '${this.githubEvent.pull_request.head.ref}' of github.com:rigup/${this.githubEvent.repository.name}`
     ];
+    const revertStart = [`Revert`];
```

Comment:
So there is less of a chance of some random commit slipping through do we want to add the jira pattern after `Revert`? 

Revert “JIRA_PATTERN”:

Reply:
Nah I think it's fine... it will filter out commit messages that start with Revert, doesn't necessarily short-circuit the check... we can always change later